CONTRIBUTING.md

```diff
@@ -111,12 +111,16 @@ function check_all_found(table)
 end
 
 function check_packages_versions(pkg_names, repo_url; registry_uuid=GENERAL_UUID, verbose=true, throw=true)
-    dir = mktempdir()
-    run(`$(git()) clone $(repo_url) $dir`)
+    if isdir(repo_url)
+        dir = repo_url
+    else
+        dir = mktempdir()
+        run(`$(git()) clone $(repo_url) $dir`)
+    end
 
     registry = only(filter!(r -> r.uuid == registry_uuid, reachable_registries()))
 
-    table = @NamedTuple{pkg_name::String, version::VersionNumber, found::Bool}[]
+    table = @NamedTuple{pkg_name::String, version::VersionNumber, found::Bool, tree_sha::Base.SHA1}[]
```

Comment:
For my personal use case my monorepo didn't have tags created for the various releases. I used the returned table from this function to lookup commit SHAs based upon the tree SHAs and create those version tags